openassessment/xblock/staff_area_mixin.py

```diff
@@ -250,6 +251,26 @@ def render_staff_grade_counts(self, data, suffix=''):  # pylint: disable=W0613
         except PeerAssessmentInternalError:
             return self.render_error(self._(u"Error getting staff grade ungraded and checked out counts."))
 
+    def get_user_email_by_submission_uuid(self, submission_uuid):
```

Comment:
Not enough clear for me. Could you slit this function it to 2 separate functions?

Reply:
Done